Ska/engarchive/tests/test_sync.py

```diff
@@ -13,6 +14,8 @@
 from .. import update_client_archive, update_server_sync
 from ..utils import STATS_DT, set_fetch_basedir
 
+pytestmark = pytest.mark.skipif(sys.maxsize <= 2 ** 32, reason="tests for 64-bit only")
```

Comment:
This looks good, though IIRC we were going to use `platform` if 32 bit tests came up again.  But maybe that was when we had less OSX interest.

Reply:
Platform docs specifically say to not use platform for this.

Reply:
I thought the issue was that it was unreliable to identify 32bit OSX.

Reply:
Do you have reason to believe that this test using sys.maxsize will NOT work in some situation?  If not, let's accept the Python docs recommendation and always do it this way.